### PR TITLE
feat(Header): Allow custom Header URL

### PR DESCRIPTION
**Pull request by athill**

### CODE REVIEW

File: src/components/Header/Header.test.tsx [hunk 1]
```diff
@@ -7,10 +7,11 @@ import * as React from 'react';
 import Header from './Header'
 import HeaderIdentity from './HeaderIdentity'
 import HeaderNavigation from './HeaderNavigation'
+import { isMainThread } from 'worker_threads';
```
Comment (burnumd) [inline src/components/Header/Header.test.tsx:10]: What is this import for? I don't see it being used.

Comment (athill) [reply src/components/Header/Header.test.tsx:10]: I have no idea where that came from. Removing. Thanks.

Comment (athill) [reply src/components/Header/Header.test.tsx:10]: Actually, I do know. {{vscode}} incorrectly auto-completed something and then auto-imported it. Thanks for catching.

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -11,6 +11,11 @@ import Identity from './HeaderIdentity';
 import Navigation from './HeaderNavigation';
 
 interface HeaderProps {
+    /**
+     * The URL that the link in the upper left of the header will point to.
+     * @see https://rivet.uits.iu.edu/components/navigation/header/
+     */
+    headerUrl: string;
```
Comment (jwbennet) [inline src/components/Header/Header.tsx:18]: My two cents is that I'd expect this to be named `href` to fall in line with the other components which have similar properties (button, etc).

Comment (athill) [reply src/components/Header/Header.tsx:18]: Agreed. Fixing.

File: src/components/Header/Header.md [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
+default view. The base header is the bare minimum version of the header that must
 be included in your application.
 
 View the [Rivet documentation for Header](https://rivet.uits.iu.edu/components/navigation/header/).
 
-
 ```jsx
 <Header title="Application Title" />
 ```
+You can also overrice the default "`/`" URL for the link:
```
Comment (burnumd) [inline src/components/Header/Header.md:11]: Typo
```suggestion
You can also override the default "`/`" URL for the link:
```

File: tsconfig.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "baseUrl": "src",
     "outDir": "build/dist",
-    "module": "commonjs",
+    "module": "esnext",
```
Comment (burnumd) [inline tsconfig.json:5]: Why did this change? It wasn't originally part of the PR.

Comment (athill) [reply tsconfig.json:5]: I don't know, but this is the second time I've reverted it. Weird. Thanks for catching.

File: src/components/Header/Header.md (hunk 3 above)
Comment (athill) [reply src/components/Header/Header.md:11]: Fixed. Thanks.

**Pull request merged**